src/cr/cube/measures/pairwise_significance.py

```diff
@@ -12,22 +12,42 @@
 try:
     xrange
 except NameError:  # pragma: no cover
-    # pylint: disable=invalid-name
     xrange = range
 
 
-class PairwiseSignificance:
+class PairwiseSignificance(object):
     """Implementation of p-vals and t-tests for each column proportions comparison."""
 
-    def __init__(
-        self, slice_, axis=0, weighted=True, alpha=0.05, only_larger=True, hs_dims=None
-    ):
+    def __init__(self, slice_, alpha=0.05, only_larger=True):
         self._slice = slice_
-        self._axis = axis
-        self._weighted = weighted
         self._alpha = alpha
         self._only_larger = only_larger
-        self._hs_dims = hs_dims
+
+    @classmethod
+    def pairwise_indices(cls, slice_, alpha, only_larger):
+        """ -> 2D ndarray of tuples of pairwise indices."""
+        return cls(slice_, alpha, only_larger)._pairwise_indices
+
+    @classmethod
+    def scale_mean_pairwise_indices(cls, slice_, alpha, only_larger):
+        """ -> 1D ndarray of tuples of column-indices meeting pairwise-t threshold.
+
+        Indicates "whole columns" that are significantly different, based on the mean of
+        the scale (category numeric-values) for each column. The length of the array is
+        that of the columns-dimension.
+        """
+        return cls(slice_, alpha, only_larger)._scale_mean_pairwise_indices
+
+    @lazyproperty
+    def summary_pairwise_indices(self):
+        """ndarray containing tuples of pairwise indices for the column summary."""
```

Comment:
considering that we're using the -> convention for the return values in the docstring I would use also here:
```
-> ndarray, tuple of pairwise ... 
```